src/evaluations/data/evaluation_configs.py

```diff
@@ -485,83 +515,278 @@ def get_evaluation_config(config_name):
     if valid_config_names[i] in valid_config_names[(i+1):]:
       duplicate_configs.append(valid_config_names[i])
   if duplicate_configs:
-    raise ValueError("Duplicate names found in evaluation configs: {}".
-                     format(','.join(duplicate_configs)))
+    raise ValueError('Duplicate names found in evaluation configs: {}'
+                     .format(','.join(duplicate_configs)))
 
   config = [c for c in configs if c().name == config_name]
   if not config:
-    raise ValueError("Invalid evaluation config: {}\n"
-                     "Valid choices are as follows: {}".format(
-                     config_name, ','.join(valid_config_names)))
+    raise ValueError('Invalid evaluation config: {}\n'
+                     'Valid choices are as follows: {}'.format(
+                         config_name, ','.join(valid_config_names)))
   return config[0]
 
 
+def _format_epsilon(epsilon=None, decimals=4):
+  """Format epsilon value to string.
+
+  Args:
+    epsilon: an optional differential private parameter. By default set to None.
+    decimals: an integer value which set the number of decimal points of the
+      epsilon to keep.
+
+  Returns:
+    A string representation of epsilon. If epsilon is set to None, will treat
+    as not adding noise and return 'infty' (infinity). Otherwise, will keep
+    decimal points set by the decimals argument.
+  """
+  if epsilon is None:
+    return INFINITY_STR
+  str_format = '{:0.' + str(decimals) + 'f}'
+  return str_format.format(float(epsilon))
+
+
+def construct_sketch_estimator_config_name(sketch_name, sketch_config,
```

Comment:
I wonder if it would be better for this to be a method of the SketchEstimator class?